cpp/util/profiler.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+#ifndef OPENAGE_UTIL_PROFILER_H_
+#define OPENAGE_UTIL_PROFILER_H_
+
+#include <map>
+#include <vector>
+#include <chrono>
+#include <list>
+
+constexpr int MAX_DURATION_HISTORY = 100;
+
+namespace openage {
+namespace util {
+
+struct CategorieTimeData {
```

Comment:
struct names are lowercased and underscored please (see the `doc/codestyle/` examples)